sqlalchemy_trino/datatype.py

```diff
@@ -142,7 +142,8 @@ def parse_sqltype(type_str: str) -> TypeEngine:
     elif type_name == "row":
         attr_types: Dict[str, SQLType] = {}
         for attr_str in split(type_opts):
-            name, attr_type_str = split(attr_str.strip(), delimiter=' ')
+            outputs = list(split(attr_str.strip(), delimiter=' '))
+            name, attr_type_str = outputs[:2]
```

Comment:
`attr_type_str` must be `timestamp(6) with time zone` in this case

Reply:
This need to be change too:
https://github.com/dungdm93/sqlalchemy-trino/blob/8045f495db4481792e7bfa865f0aad3699b89860/sqlalchemy_trino/datatype.py#L138

Reply:
ok, will fix, thx!